packages/styles/scss/components/search/_search.scss

```
@@ -0,0 +1,368 @@
+//
+// Copyright IBM Corp. 2016, 2018
+//
+// This source code is licensed under the Apache-2.0 license found in the
+// LICENSE file in the root directory of this source tree.
+//
+
+//-----------------------------
+// Search
+//-----------------------------
+
+@use '../../theme' as *;
+@use '../../config' as *;
+@use '../../utilities/placeholder-colors' as *;
+@use '../../utilities/focus-outline' as *;
+@use '../../utilities/visually-hidden' as *;
+@use '../../motion' as *;
+@use '../../spacing' as *;
+@use '../../utilities/component-reset' as *;
+@use '../../utilities/skeleton' as *;
+@use '../../utilities/high-contrast-mode' as *;
+@use '../../type';
```

If there's no mixin name conflicts you can use the `as *`

```suggestion
@use '../../type' as *;
```

which then simplifies the usages:

```diff
-@include type.type-style('body-short-01');
+@include type-style('body-short-01');
```

Also a small style nit I got from Josh: order of imports should be:

* top-level (colors, config, etc)
* component imports (form, others used)
* utilities
* local

---

Ah, yeah I was getting more specific with type because both /type and /component-reset forwarded `@include reset` which was throwing an error

---

If there's a better way to take care of those overlapping imports like that let me know tho

---

No that's totally cool, I ran into the same thing!